src/video/cocoa/cocoa_wnd.mm

```diff
@@ -807,12 +1104,17 @@ - (void)doCommandBySelector:(SEL)aSelector
 @end
 
 
+@implementation OTTD_CocoaWindowDelegate {
+	VideoDriver_Cocoa *driver;
+}
 
-@implementation OTTD_CocoaWindowDelegate
 /** Initialize the video driver */
-- (void)setDriver:(CocoaSubdriver*)drv
+- (instancetype)initWithDriver:(VideoDriver_Cocoa *)drv
 {
-	driver = drv;
+	if (self = [ super init ]) {
```

Comment:
I suppose the `=` is intentional? Could use (()) to make it more obviously intentional?

Reply:
Common Objective-C ideom and usually written like this. I can add `()`s if necessary.